integrations/nielsen-dcr/lib/index.js

```diff
@@ -165,18 +165,19 @@ NielsenDCR.prototype.heartbeat = function(assetId, position, options) {
 
 NielsenDCR.prototype.getContentMetadata = function(track, type) {
   var properties = 'properties.';
```

Comment:
I think the variable name here is a bit confusing as `properties` is usually used to indicate an object containing the event properties. Would it make sense to name this something like `propertiesPath`?